### ISSUE TITLE
Possible performance issue

### ISSUE THREAD

**Issue by bradphelan** [Weingartner/FixNuGetPackagePath#5]
When I update a single nuget package. ie: devDept.Eyeshot I see the following in the output window

> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project WeinCad.Plugin.GearCorrection ======
> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project WeinCad.Plugin.GearCorrection.Spec ======
> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project WeinCad.Plugin.FlankEvaluation ======
> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project WeinCad.Plugin.MoineauPointList.Spec ======
> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project WeinCad.Plugin.Peeling.Specs ======
> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project WeinCad.Plugin.PointListValidation ======
> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project WeinCad.Plugin.BasicFolders.Spec ======
> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project WeinCad.Plugin.ExtruderMill ======
> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project WeinCad.DataMigration.Spec ======
> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project Weingartner.Wpf.Common ======
> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project Weingartner.Wpf.Animation ======
> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project Weingartner.DotLiquidExtensions.Spec ======
> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project Weingartner.DotLiquidExtensions ======
> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project QuickGraph ======
> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project LaserHelpers ======
> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project KeyenceCom ======
> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project LaserHelpers.Spec ======
> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project WeinCadOldImport ======
> [Info] Fixed 0 paths.
> [Info] ===== Fix paths for NuGet package devDept.Eyeshot in project WeinCadOldImport.Spec ======
> [Info] Fixed 0 paths.

Which goes on for thousands of lines.  This is a huge amout of output considering that it should really only be run once. Looks like it's responding immediately to every change nuget makes.  I'm not sure if this is impacting the overall shitty amount of time that nuget takes to upgrade a package but making it even a bit worse certainly doesn't make it better.

**Comment by johannesegger** [comment]
Yep, and "Fixed 0 paths" might be useless information anyway.

**Comment by bradphelan** [comment]
I think this problem will go away once the Nuget team figures out how to fix the disaster that is PackageReference. The hint paths are no longer needed. However I just spent a day figuring out that the much hyped feature is really only in alpha and you should stay away from it unless maybe you are only targetting .Net Core/Standard

**Comment by johannesegger** [comment]
Haven't tried that yet. I'm a bit frustrated with PCL/.Net Standard/.Net Core because there are always problems.

**Comment by bradphelan** [comment]
Yes ... I think the problems are to be summed up by this comment/rant I found.

http://disq.us/p/1habbuz

**Comment by johannesegger** [comment]
Ohh, I guess I wouldn't want to be on the NuGet team right now, they seem to have a hard time explaining the crap they made.
